docs/src/docker.rst

```diff
@@ -0,0 +1,41 @@
+=============
+ Docker
+=============
+
+**cijoe** can run in docker. First, pull the docker image from DockerHub or
+build it yourself.
+
+.. code-block:: bash
+
+  # Pull cijoe image
+  docker pull cijoe/cijoe
+
+.. code-block:: bash
+
+  # Build it yourself
+  docker build . -t cijoe/cijoe
+
+.. note:: **cijoe** will force the **mktemp** command to create temporary
+  directories in the /results folder. This allows for easy extraction of test
+  results, but may cause issues if your application relies on **mktemp**. We
+  will resolve this issue in a future release.
+
+You can now start the docker container and run a testplan. To make running
+testplans easier, you can link your target environment and results folder.
+
+.. code-block:: bash
+
+  # Run cijoe
+  docker run -it \
+  -v $(pwd)/target_env.sh:/cijoe/target_env.sh \
+  -v $(pwd)/results:/results \
+  cijoe
+
+You can now run your testplan. See `QuickStart`_ for examples on how to use
+**cijoe**. Starting the docker container is equivalent to running the **cijoe**
+command.
+
+.. note:: If you run **cijoe** on a remote system, bear in mind that you may
```

Comment:
Would be great with documentation on how to transfer keys and manage populate the ssh-agent inside the container.

Reply:
Alright. I have added a small feature that eases the pain of adding the keys to the docker container. It is not so easy to do for others due to the strict permission requirements. I have made the feature such that you can mount your SSH folder to `/tmp/cijoe/.ssh`, and then cijoe will do the rest.

I have also added a warning to the docs saying that this is insecure with untrused code, as the software has full access to the private key.